freqtrade/rpc/rpc.py

```diff
@@ -328,6 +328,16 @@ def _rpc_reload_conf(self) -> Dict[str, str]:
         self._freqtrade.state = State.RELOAD_CONF
         return {'status': 'reloading config ...'}
 
+    def _rpc_stopbuy(self) -> Dict[str, str]:
+        """
+        Handler to stop buying, but handle open trades gracefully.
+        """
+        if self._freqtrade.state == State.RUNNING:
+            # Set 'max_open_trades' to 0
+            self._freqtrade.config['max_open_trades'] = 0
+
+        return {'status': 'Setting max_open_trades to 0. Run /reload_conf to reset.'}
```

Comment:
this message doesn't have to be technical in my opinion. it can be "No more buy will occur from now. to resume buys run /reload_conf."